superset-frontend/packages/superset-ui-core/src/utils/featureFlags.ts

```diff
@@ -44,6 +44,7 @@ export enum FeatureFlag {
   ENABLE_ADVANCED_DATA_TYPES = 'ENABLE_ADVANCED_DATA_TYPES',
   ENABLE_EXPLORE_DRAG_AND_DROP = 'ENABLE_EXPLORE_DRAG_AND_DROP',
   ENABLE_JAVASCRIPT_CONTROLS = 'ENABLE_JAVASCRIPT_CONTROLS',
+  ENABLE_TELEMETRY = 'ENABLE_TELEMETRY',
```

Comment:
Can we remove this now that we depend on an environment variable?

Reply:
LOL you're on top of it! Done, and more cleanup to come.